src/catchup/ApplyBucketsWork.cpp

```diff
@@ -126,28 +126,20 @@ ApplyBucketsWork::onStart()
         CLOG(DEBUG, "History") << "ApplyBuckets : starting level[" << mLevel
                                << "].curr = " << i.curr;
         mApplying = true;
-        mBucketApplyStart.Mark();
+        mBucketApplyStart.inc();
     }
 }
 
 void
 ApplyBucketsWork::onRun()
 {
-    try
```

Comment:
This doesn't belong to this commit

Reply:
Right. I'll update that when I'll be doing rebases when it is accepted.